src/components/Select/components/SelectList/SelectList.scss

```diff
@@ -42,29 +32,34 @@ $xl-right-padding: '12px';
         padding: 0;
 
         #{$block}_size_s &:not(#{&}_empty) {
-            height: #{$s-height};
-            padding: 8px #{$s-right-padding} 4px #{$s-left-padding};
+            height: #{variables.$s-height};
+            padding: #{select-css-variables.$s-hor-padding} #{select-css-variables.$s-hor-padding}
+                4px;
         }
 
         #{$block}_size_m &:not(#{&}_empty) {
-            height: #{$m-height};
-            padding: 8px #{$m-right-padding} 4px #{$m-left-padding};
+            height: #{variables.$m-height};
+            padding: #{select-css-variables.$m-hor-padding} #{select-css-variables.$m-hor-padding}
```

Comment:
Using same var for both top and side padding can be misleading for other contributors